### ISSUE TITLE
Unexpected grouping when using compose project and group labels

### ISSUE THREAD

**Issue by arjoonn-s** [amir20/dozzle#3069]
**Is your feature request related to a problem? Please describe.**
I'm not sure if this is a feature / bug or just a generic question.

**Describe the solution you'd like**

- I have a compose file that describes api, worker, database containers, and dozzle itself.
- api and workers have replicas > 1. The rest do not specify replicas.
- I've added labels `dev.dozzle.group=api` and `dev.dozzle.group=worker` to the api and worker containers respectively.
- I was expecting that I'll be able to combine api logs and worker logs from all their respective replicas but so far I'm only able to combine the entire project.
- ![image](https://github.com/amir20/dozzle/assets/120045237/549f4cd7-69b4-45ac-a1ea-c615f40e7288)

I'm not sure what I'm doing wrong or if this can even be considered something in scope for dozzle?

PS: I love this project! I'm using v8.0.1

**Comment by amir20** [comment]
Hmmm you should be seeing a `Swarm Mode` when you logs to be merged. This can happen with custom labels or swarm labels. 

Let me test it to see if I broke something.

**Comment by amir20** [comment]
It works on my side. I created two instance of a test container: 

```
docker run -d --label dev.dozzle.group=test amir20/echo -r
docker run -d --label dev.dozzle.group=test amir20/echo -r
```

Then I see swarm mode. See video: 


https://github.com/amir20/dozzle/assets/260667/167fcd6a-b04a-49b8-b9cd-14cb66f54b3c

**Comment by amir20** [comment]
And holy cow 😂 32 CPUs. What monster are you running?

**Comment by amir20** [comment]
I wish everybody would provide a yml file for me to reproduce. 🙏

> Hmm. I haven't tried swarm mode 🤔 . I'll give it a go in the morning tomorrow.

There might be confusion here. Swarm mode in the UI has nothing to do with using swarm. It just merges the logs in to one stream when there is any kind of grouping, eg. services or custom groups. 

I used your file and everything worked fine. 

<img width="1658" alt="Screenshot 2024-07-06 at 12 22 23 PM" src="https://github.com/amir20/dozzle/assets/260667/cf91c118-e91a-4653-b9d4-d46d65e68424">

Again, note that swarm mode shows up automatically when the UI sees a grouping. For **some reason** it is not there for you. 

I used `docker compose -f bug-3069.yml up` to bring it up. 

So something is obviously different in your environment. Based on your screenshot I can't tell what it is.  

I am not sure how to proceed? 😅
